iceoryx_dust/include/iceoryx_dust/internal/cxx/static_storage.hpp

```diff
@@ -14,8 +14,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef IOX_HOOFS_STATIC_STORAGE_HPP
-#define IOX_HOOFS_STATIC_STORAGE_HPP
+#ifndef IOX_DUST_STATIC_STORAGE_HPP
+#define IOX_DUST_STATIC_STORAGE_HPP
 
```

Comment:
@FerdinandSpitzschnueffler We may consider fixing the last 2 remaining Axivion warnings right here, I can add a commit myself. It would be the C array usage in this file (questionable rule but OK).